pkg/v1/tkg/client/validate.go

```diff
@@ -1800,3 +1806,31 @@ func (c *TkgClient) validateNameservers(nameserverConfigVariable string) error {
 	}
 	return nil
 }
+
+func (c *TkgClient) checkIPFamilyFeatureFlags(ipFamily, clusterRole string) error {
+	if clusterRole == TkgLabelClusterRoleManagement {
+		dualIPv4PrimaryEnabled := c.IsFeatureActivated(config.FeatureFlagManagementClusterDualStackIPv4Primary)
+		if !dualIPv4PrimaryEnabled && ipFamily == constants.DualStackPrimaryIPv4Family {
+			return dualStackFeatureFlagError(ipFamily, config.FeatureFlagManagementClusterDualStackIPv4Primary)
+		}
+		dualIPv6PrimaryEnabled := c.IsFeatureActivated(config.FeatureFlagManagementClusterDualStackIPv6Primary)
+		if !dualIPv6PrimaryEnabled && ipFamily == constants.DualStackPrimaryIPv6Family {
+			return dualStackFeatureFlagError(ipFamily, config.FeatureFlagManagementClusterDualStackIPv6Primary)
+		}
+	} else {
+		dualIPv4PrimaryEnabled := c.IsFeatureActivated(config.FeatureFlagClusterDualStackIPv4Primary)
+		if !dualIPv4PrimaryEnabled && ipFamily == constants.DualStackPrimaryIPv4Family {
+			return dualStackFeatureFlagError(ipFamily, config.FeatureFlagClusterDualStackIPv4Primary)
+		}
+		dualIPv6PrimaryEnabled := c.IsFeatureActivated(config.FeatureFlagClusterDualStackIPv6Primary)
+		if !dualIPv6PrimaryEnabled && ipFamily == constants.DualStackPrimaryIPv6Family {
+			return dualStackFeatureFlagError(ipFamily, config.FeatureFlagClusterDualStackIPv6Primary)
+		}
+	}
+
+	return nil
+}
+
+func dualStackFeatureFlagError(ipFamily, featureFlag string) error {
+	return fmt.Errorf("option TKG_IP_FAMILY is set to %q, but dualstack support is under development. To enable dualstack, set %s to \"true\"", ipFamily, featureFlag)
```

Comment:
I would tweak this message slightly to:
"option TKG_IP_FAMILY is set to %q, but dualstack support is not enabled (because it is under development)..."

Reply:
Updated! Thanks for the suggestion!